packages/scripts/network-setup/deployLib.ts

```diff
@@ -110,6 +135,24 @@ export const deployTeritoriEcosystem = async (
     network,
   );
 
+  console.log("Instantiating NFT Launchpad", network.nameServiceCodeId);
+  const cosmwasmLaunchpadFeature = cloneDeep(
+    getNetworkFeature(networkId, NetworkFeature.NFTLaunchpad),
+  );
+  if (!cosmwasmLaunchpadFeature) {
+    console.error(`Cosmwasm Launchpad feature not found on ${networkId}`);
+  } else {
+    cosmwasmLaunchpadFeature.launchpadContractAddress =
+      await instantiateNftLaunchpad(
+        opts,
+        wallet,
+        walletAddr,
+        "TODO DAO address",
```

Comment:
is it mean to be done in this PR or in another PR?, would be nice to open issue if it will not be done in this PR

Reply:
This script is used to deploy contracts. This is the way we currently should use to deploy and instantiate contracts.

Hmm the admin DAO address can be set in the args when executing the script (`npx tsx deployNftLaunchpad.ts`).
I'll remove `deployNftLaunchpad` usage from `deployTeritoriEcosystem` to avoid adding an arg to `deployTeritoriEcosystem`..
==> If we want to use NFT Launchpad scripts, we use it as standalone.